### PR TITLE
Updated dataset in Sentiment Analysis

### LINKED ISSUE #7024: Microsoft.ML sentiment-analysis results issue with different versions of Microsoft.ML
Seems to be an issue with this sample

The sample works with Nuget Package Microsoft.ML 0.3.0
When running 0.4.0 you get different results.

That is, both results in the console are positive for 0.4.0. The first result should be negative if it is supposed to be the same as 0.3.0, which I'd assume it would.

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c8e22825-c4ee-5a36-f911-8ad456970ecc
* Version Independent ID: ad0bf640-7dfc-1184-077c-358bb14277a7
* Content: [Use ML.NET in a sentiment analysis binary classification scenario](https://docs.microsoft.com/en-au/dotnet/machine-learning/tutorials/sentiment-analysis)
* Content Source: [docs/machine-learning/tutorials/sentiment-analysis.md](https://github.com/dotnet/docs/blob/master/docs/machine-learning/tutorials/sentiment-analysis.md)
* Product: **dotnet-ml**
* GitHub Login: @JRAlexander
* Microsoft Alias: **johalex**

### LINKED ISSUE #10080: Why?
@frgt10cs commented on [Sat Jan 19 2019](https://github.com/dotnet/docs.ru-ru/issues/164)

Я делал всё по документации, код полностью идентичен (даже копировал-вставлял), в итоге результаты не сходятся. Ваши выходные данные:
Sentiment: This is a very rude movie | Prediction: Toxic | Probability: 0.4585565
Sentiment: He is the best, and the article should say that. | Prediction: Not Toxic | Probability: 0.9924279
Мои:
Sentiment: This is a very rude movie | Prediction: Toxic | Probability: 0,5297049
Sentiment: He is the best, and the article should say that. | Prediction: Toxic | Probability: 0,9918675
Что не так? Почему различие?

---
#### Сведения о документе

⚠ *Не вносите правки в этот раздел. Это необходимо для связывания страницы сайта docs.microsoft.com с вопросом на GitHub.*

* ID: c8e22825-c4ee-5a36-f911-8ad456970ecc
* Version Independent ID: ad0bf640-7dfc-1184-077c-358bb14277a7
* Content: [Использование ML.NET для анализа тональности методом двоичной классификации](https://docs.microsoft.com/ru-ru/dotnet/machine-learning/tutorials/sentiment-analysis#machine-learning-workflow)
* Content Source: [docs/machine-learning/tutorials/sentiment-analysis.md](https://github.com/dotnet/docs.ru-ru/blob/live/docs/machine-learning/tutorials/sentiment-analysis.md)
* Product: **dotnet-ml**
* GitHub Login: @JRAlexander
* Microsoft Alias: **johalex**

### LINKED ISSUE #10106: Incorrect Toxic/NonToxic Prediction Value
After going through the documents for the text sentiment ML I wasn't getting the expected results as stated in the docs.  After a bit more testing I downloaded the actual files and came out with the same result.

> Sentiment: This is a very rude movie | Prediction: Toxic | Probability: 0.5297049
> Sentiment: He is the best, and the article should say that. | Prediction: Toxic | Probability: 0.9918675

These two lines are both predicting that the sentiments are "Toxic" however based on the docs and what 
should naturally be expected the second line should be considered "Non-Toxic" which is being shown as "Toxic" instead.

The below result is what the expectation would be if working correctly.

> Sentiment: This is a very rude movie | Prediction: Toxic | Probability: 0.4585565
> Sentiment: He is the best, and the article should say that. | Prediction: Not Toxic | Probability: 0.9924279
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c8e22825-c4ee-5a36-f911-8ad456970ecc
* Version Independent ID: ad0bf640-7dfc-1184-077c-358bb14277a7
* Content: [Use ML.NET in a sentiment analysis binary classification scenario](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sentiment-analysis)
* Content Source: [docs/machine-learning/tutorials/sentiment-analysis.md](https://github.com/dotnet/docs/blob/master/docs/machine-learning/tutorials/sentiment-analysis.md)
* Product: **dotnet-ml**
* GitHub Login: @JRAlexander
* Microsoft Alias: **johalex**

### LINKED ISSUE #11054: M35452: Wrong formatting in Source on Website
Hello @JRAlexander, @BillWagner, @mairaw, @rpetrusha,
Translator has reported possible source content issue.
Link to Github file: https://github.com/dotnet/docs/blob/master/docs/machine-learning/tutorials/sentiment-analysis.md
Row where the issue is located: 295
Comment: Please note that something went wrong with the formatting on the Website. 

Can you please review the comment and provide answer/explanation? Also, if source fix is needed please provide link to the PR that will include the fix, so we can confirm and close this Issue.
Many thanks in advance.

### PR DESCRIPTION
**Pull request by JRAlexander**
Updated dataset from wikidetox to UCI Sentiment in Sentiment Analysis tutorial to improve model prediction, and updated to version 0.11.0.

[Related Code Sample](https://github.com/dotnet/samples/pull/680)

[Internal Review Link](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sentiment-analysis?branch=pr-en-us-11114)

Fixes https://github.com/dotnet/docs/issues/7024 
Fixes https://github.com/dotnet/docs/issues/10080 
Fixes https://github.com/dotnet/docs/issues/10106
Fixes https://github.com/dotnet/docs/issues/11054

### CODE REVIEW

File: docs/machine-learning/tutorials/sentiment-analysis.md [hunk 1]
```diff
@@ -319,73 +343,73 @@ The `Predict` method executes the following tasks:
...
+Now that you have a model, you can use that to predict the Toxic or Non Toxic sentiment of the comment data using the <xref:Microsoft.ML.Core.Data.ITransformer.Transform%2A> method. To get a prediction, use `Predict` on new data. Note that the input data is a string and the model includes the featurization. Your pipeline is in sync during training and prediction. You didn’t have to write preprocessing/featurization code specifically for predictions, and the same API takes care of both batch and one-time predictions. Add the following code to the `UseLoadedModelWithBatchItems` method for the predictions:
 
-[!code-csharp[Predict](../../../samples/machine-learning/tutorials/SentimentAnalysis/Program.cs#28 "Create predictions of sentiments")]
+[!code-csharp[Predict](../../../samples/machine-learning/tutorials/SentimentAnalysis/Program.cs#Prediction "Create predictions of sentiments")]
 
 ### Using the loaded model for prediction
```
Comment (luisquintanilla) [inline docs/machine-learning/tutorials/sentiment-analysis.md]: Change tense. Use the lodaded model for prediction

File: docs/machine-learning/tutorials/sentiment-analysis.md [hunk 2]
```diff
@@ -319,73 +343,73 @@ The `Predict` method executes the following tasks:
...
+ You can use that to predict the positive or negative sentiment of a single instance of the comment data. To get a prediction, use <xref:Microsoft.ML.PredictionEngine%602.Predict%2A> on the data. Note that the input data is a string and the model includes the featurization. Your pipeline is in sync during training and prediction. You didn’t have to write preprocessing/featurization code specifically for predictions, and the same API takes care of both batch and one-time predictions.
 
-[!code-csharp[Predict](../../../samples/machine-learning/tutorials/SentimentAnalysis/Program.cs#19 "Create a prediction of sentiment")]
+[!code-csharp[Predict](../../../samples/machine-learning/tutorials/SentimentAnalysis/Program.cs#Predict "Create a prediction of sentiment")]
 
 ### Using the model: prediction
```
Comment (luisquintanilla) [inline docs/machine-learning/tutorials/sentiment-analysis.md]: Change tense. Use the model for prediction

**Review by luisquintanilla — CHANGES_REQUESTED** [review]
Nice work @JRAlexander . Minor changes needed.

File: docs/machine-learning/tutorials/sentiment-analysis.md [hunk 3]
```diff
@@ -134,38 +136,73 @@ You need to create some classes for your input data and predictions. Add a new c
...
+```csharp
+public static (IDataView trainSet, IDataView testSet) LoadData(MLContext mlContext)
+{
 
-The `Train` method executes the following tasks:
+}
+```
+## Load the data
+
+Since your previously created `SentimentData` data model type matches the dataset schema, you can combine the initialization, mapping, and dataset loading into one line of code using the `MLContext.Data.ReadFromTextFile` wrapper for <xref:Microsoft.ML.TextLoaderSaverCatalog.ReadFromTextFile%60%601>. It returns a
```
[the remark this answers is not in the document]
Comment (JRAlexander) [reply docs/machine-learning/tutorials/sentiment-analysis.md]: Yeah thats exactly it. I think I'm just going to update this for .11 and be done with it.

**Comment by JRAlexander** [comment]
Thanks, @luisquintanilla! I believe I've addressed your concerns.

File: docs/machine-learning/tutorials/sentiment-analysis.md [hunk 4]
```diff
@@ -105,26 +107,29 @@ Classification algorithms are frequently one of the following types:
 
 ### Prepare your data
 
-1. Download the [Wikipedia detox-250-line-data.tsv](https://github.com/dotnet/machinelearning/blob/master/test/data/wikipedia-detox-250-line-data.tsv) and the [wikipedia-detox-250-line-test.tsv](https://github.com/dotnet/machinelearning/blob/master/test/data/wikipedia-detox-250-line-test.tsv) data sets and save them to the *Data* folder previously created. The first dataset trains the machine learning model and the second can be used to evaluate how accurate your model is.
+1. Download [The UCI Sentiment Labeled Sentences dataset zip file (see citations in the following note)](https://archive.ics.uci.edu/ml/machine-learning-databases/00331/sentiment%20labelled%20sentences.zip), and unzip.
+
+2. Copy the `yelp_labeled.txt` file into the *Data* directory you created.
```
Comment (luisquintanilla) [inline docs/machine-learning/tutorials/sentiment-analysis.md]: Change file name to match the name in the code sample -> yelp_labelled.txt

**Review by luisquintanilla — CHANGES_REQUESTED** [review]
Thanks for making the changes. One last change. Update the name of the data file to match that of the code sample yelp_labelled.txt.

File: docs/machine-learning/tutorials/sentiment-analysis.md (hunk 4 above)
Comment (JRAlexander) [reply docs/machine-learning/tutorials/sentiment-analysis.md]: Good catch.

**Review by luisquintanilla — APPROVED** [review]

**Pull request merged**